src/Microsoft.ML.Data/DataView/Transposer.cs

```diff
@@ -810,27 +810,48 @@ public DataViewSlicer(IHost host, IDataView input, int[] toSlice)
                 {
                     var splitter = _splitters[c] = Splitter.Create(_input, toSlice[c]);
                     _host.Assert(splitter.ColumnCount >= 1);
+                    // One splitter can produce multiple columns because it split a input column into multiple output columns.
```

Comment:
>split a [](start = 76, length = 7)

nit: splits an

Reply:
Fixed.

---
In reply to: [246276306](https://github.com/dotnet/machinelearning/pull/2080#discussion_r246276306) [](ancestors = 246276306)